modules/spring-config/src/main/java/org/mule/runtime/config/internal/context/AbstractSpringMuleContextServiceConfigurator.java

```diff
@@ -49,6 +69,13 @@ protected AbstractSpringMuleContextServiceConfigurator(InternalCustomizationServ
     this.serviceLocator = serviceLocator;
   }
 
+  protected void registerContextServices(Map<String, BeanDefinition> contextServices, ArtifactType artifactType) {
+    contextServices.entrySet().stream()
+        .filter(service -> !APPLICATION_ONLY_SERVICES.contains(service.getKey()) || artifactType.equals(APP)
```

Comment:
artifactType is a parameter of the registerContextServices method. Are these two equals ok to be here?